setup.py

```diff
@@ -28,6 +28,6 @@
                    'Programming Language :: Python',
                    'Framework :: Django'],
     packages = find_packages(),
-    setup_requires=[],
+    install_requires=['Django>=3'],
```

Comment:
Just Django 3, or should we force Django 4?

Reply:
Just Django 3 since the code is still compatible with it which means we can merge this in without all the games needing to be using Django 4 right away